modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -78,7 +78,21 @@ above will no longer run in IE11 and will fail to bootstrap with an error and/or
 
 ### React 18 Upgrade
 
-> Coming Soon (https://github.com/Workday/canvas-kit/discussions/1545)
+In this release we upgraded React to version 18. [React 18](https://reactjs.org/blog/2022/03/29/react-v18.html) introduced several new features, but the
+most impactful change is the new [concurrent render API](https://reactjs.org/blog/2022/03/29/react-v18.html#what-is-concurrent-react). In the new API, you'll use the `.createRoot()`
+method which replaces `.render()`, which adds all of the improvements of React 18 and allows you to
+use concurrent features. Without it React behaves the same as it did in version 17.
+
+Canvas Kit doesn’t use the render API directly in our packages but Storybook uses it to render
+Canvas Kit components. Because of this, we upgraded Storybook dependencies to support React 18.
+
+**Impact on CK:** Updating to React 18 didn't cause any breaking changes in our code. However, we did have to update some types based on some type packages that where upgraded.
+To see React v18 features in action we had to upgrade Storybook to v6.5.x.
```

Comment:
Thoughts on removing the **Impact on CK** section? I'm not sure we need it in the Upgrade Guide. If we feel it's important, we could include a link to the [GH discussion](https://github.com/Workday/canvas-kit/discussions/1545).